### PR TITLE
test: content discovery/fetching test plan

### PR DESCRIPTION
**Pull request by matthias-wright**
## Why
See https://github.com/fleek-network/ursa/issues/271. This allows us to benchmark how fast nodes can retrieve content from their peers using bitswap.

## What
- Added new test plan to test and benchmark bitswap retrieval from peers.

## Checklist

- [N/A] I have made corresponding changes to the tests
- [N/A] I have made corresponding changes to the documentation
- [x] I have run the app using my feature and ensured that no functionality is broken

### CODE REVIEW

File: test-plans/fetching/manifest.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+builder = "docker:generic"
+runner = "local:docker"
+
+[builders."docker:generic"]
+enabled = true
+
+[runners."local:docker"]
+enabled = true
+
+[[testcases]]
```
Comment (kckeiks) [inline test-plans/fetching/manifest.toml:13]: What is this table for? Can we control which tests run with this?

Comment (kckeiks) [reply test-plans/fetching/manifest.toml:13]: I was thinking last night if we might be able to access this value in the code, then we can control which tests run and we might not need to split tests. What do you think?

Comment (matthias-wright) [reply test-plans/fetching/manifest.toml:13]: If there are multiple binaries within one test plan, they are listed there.

Comment (matthias-wright) [reply test-plans/fetching/manifest.toml:13]: For each binary, we can specify different test cases and also specify parameters for those test cases.

**Review by kckeiks — APPROVED** [review]
lgtm

**Pull request merged**